powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/replay/ReplayVerificationService.java

```diff
@@ -41,6 +42,10 @@
  */
 @Service
 @Slf4j
+@ConditionalOnProperty(
```

Comment:
Why not. Personally, I would prefer a solution with NoOpt implementation annotated something like `@ConditionalOnMissingBean(ReplayVerificationService.class)`. It helps us to avoid null-checks.